plugins/database/src/general_query.cpp

```diff
@@ -1683,7 +1701,16 @@ genqAppendAccessCheck() {
             cllBindVars[cllBindVarCount++] = sessionTicket;
             cllBindVars[cllBindVarCount++] = sessionTicket;
             cllBindVars[cllBindVarCount++] = sessionTicket;
-            if ( !rstrcat( whereSQL, "( R_DATA_MAIN.data_id in (select object_id from R_TICKET_MAIN TICK where TICK.ticket_string=?) OR R_COLL_MAIN.coll_id in (select object_id from R_TICKET_MAIN TICK where TICK.ticket_string=?) OR R_COLL_MAIN.coll_name LIKE (select (coll_name || '%') from R_COLL_MAIN where coll_id in (select object_id from R_TICKET_MAIN TICK where TICK.ticket_string=?)))", MAX_SQL_SIZE_GQ ) ) { return USER_STRLEN_TOOLONG; }
+            if (!rstrcat(whereSQL,
+                         "R_COLL_MAIN.coll_id in (select object_id from R_OBJT_ACCESS OA, R_USER_GROUP UG, R_USER_MAIN "
+                         "UM, R_TOKN_MAIN TM where UM.user_name=? and UM.zone_name=? and "
+                         "UM.user_type_name!='rodsgroup' and UM.user_id = UG.user_id and UG.group_user_id = OA.user_id "
+                         "and OA.object_id = R_COLL_MAIN.coll_id and OA.access_type_id >= TM.token_id and "
+                         "TM.token_namespace ='access_type' and TM.token_name = 'read_object')",
+                         MAX_SQL_SIZE_GQ))
```

Comment:
This breaks tickets. Please revert this hunk and I think the rest should be good to go.

Reply:
but it's just a space -> underscore change?    discush!

Reply:
oh... no!   it's not just an underscore...   have i copy/pasted clang-format incorrectly?!

Reply:
appears so!   hold please.

Reply:
okay, repushed - will await the great clang-format once more.     apologies.